examples/wait-for-file-download/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Remove this file.
Change the export for the `path.join` method instead of turning off the `no-shadow` rule .
```js
import { join: joinPath } from 'path`;
```